kbase-extension/static/kbase/js/kbaseNarrative.js

```diff
@@ -404,34 +402,25 @@ define([
     };
 
     /**
-     * Expects the usual workspace object info array. If that's present, it's captured. If not,
-     * we run get_object_info_new and fetch it ourselves. Note that it should have its metadata.
+     * Fetches the current narrative ref info from the Workspace.
+     * @returns Promise that resolves when the latest narrative object info has been
+     * retrieved.
      */
-    Narrative.prototype.updateDocumentVersion = function (docInfo) {
-        const self = this;
-        return Promise.try(() => {
-            if (docInfo) {
-                self.documentVersionInfo = docInfo;
-            } else {
-                const workspace = new Workspace(Config.url('workspace'), {
-                    token: self.getAuthToken(),
+    Narrative.prototype.updateDocumentVersion = function () {
```

Comment:
The passed in `docInfo` parameter wasn't used by any other calling function, so apparently it's not needed.

This also restructures the Promise chain a bit and actually returns when it's done. Possibly another source of the error.